pages/learn/develop/runtime.md

```diff
@@ -12,7 +12,7 @@ For many applications, the code running in your container will need some way to
 
 ### Environment variables
 
-Inside your running container, you'll have access to a number of `{{ $names.company.allCaps }}_` namespaced environment variables, which provide information from the system outside the container:
+Inside your running container, you'll have access to a number of `{{ $names.company.allCaps }}_` namespaced environment variables, which provide information from the system outside the container:P
```

Comment:
```suggestion
Inside your running container, you'll have access to a number of `{{ $names.company.allCaps }}_` namespaced environment variables, which provide information from the system outside the container:
```

Reply:
@chrisys Its done, no idea how that one got in. Thanks for maintaining balena!